components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -116,14 +116,9 @@ export default function () {
     }
 
     const formatDate = (date: string | undefined) => {
-        return date ? moment(date).fromNow() : "–";
+        return date ? moment(date).fromNow() : "";
     }
 
-    const formatHash = (longHash: string) => {
-        return longHash ? longHash.substring(0, 8) : "–";
-    }
-
-
     return <>
         <Header title="Prebuilds" subtitle={`View all recent prebuilds for all active branches.`} />
         <div className="lg:px-28 px-10">
```

Comment:
**question**: What's the benefit of disabling the _Trigger Prebuild_ button?

Reply:
**suggestion**: What do yuou think of using a confirmation modal to run a prebuild?

> **Title**: Trigger Prebuild
> **Paragraph**: Are you sure you want to trigger a new prebuild for the default branch for this project?
> **Actions**: Cancel, Trigger Prebuild

Reply:
> issue: Button click now triggers a prebuild but it does not show up in the list unless you reload the page. Expected?

@gtsiolis, this is a known limitation for now. I'm currently working on eventing, so that we get prebuild event to the frontend. similar to the workspace instance updates, which would allow us to immediately render a state. 

as of now reloading will fetch the state a single time only.